config/cache.php

```diff
@@ -98,6 +95,6 @@
 
 	'prefix' => env(
 		'CACHE_PREFIX',
-		Str::slug(env('APP_NAME', 'laravel'), '_') . '_cache'
+		Str::slug(env('APP_NAME', 'Lychee'), '_') . '_cache'
```

Comment:
I don't know if the capitalisation is significant here, but it's worth considering.